plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -491,11 +491,12 @@ def _onCloudPrintingConfigured(self) -> None:
                 active_machine.setMetaDataEntry("do_not_show_cloud_message", True)
             return
 
-    def _onDontAskMeAgain(self, messageId: str) -> None:
+    def _onDontAskMeAgain(self, checked: bool) -> None:
         active_machine = self._application.getMachineManager().activeMachine # type: Optional["GlobalStack"]
         if active_machine:
-            active_machine.setMetaDataEntry("do_not_show_cloud_message", True)
-            Logger.log("d", "Will not ask the user again to cloud connect for current printer.")
+            active_machine.setMetaDataEntry("do_not_show_cloud_message", checked)
```

Comment:
It might be a good idea to start putting some of these magic strings behind constants. Less chance of typos because Mypy could then help check them.